Source/Particles/MultiParticleContainer.cpp

```diff
@@ -1138,15 +1147,9 @@ MultiParticleContainer::doQEDSchwinger ()
         if (!box.intersects(global_schwinger_box)) {continue;}
         box &= global_schwinger_box;
 
-        const auto& arrEx = Ex[mfi].array();
-        const auto& arrEy = Ey[mfi].array();
-        const auto& arrEz = Ez[mfi].array();
-        const auto& arrBx = Bx[mfi].array();
-        const auto& arrBy = By[mfi].array();
-        const auto& arrBz = Bz[mfi].array();
-
-        const Array4<const amrex::Real> array_EMFAB [] = {arrEx,arrEy,arrEz,
```

Comment:
In the original implementation, it's also not fully clear/defined what NVCC does when you pass a C-array to a lambda. C-arrays are captured by reference - which is not possible for host-device transfers - or by pointer - which would be a host-pointer.

It's likely that CUDA wraps around this and spots this properly, but at least for NVCC 10.0.130 this does not Work. Anyway, we can make this more clear by using a named struct, which also helps self-documenting the code :)